lib/diaspora/taggable.rb

```diff
@@ -58,5 +58,21 @@ def self.format_tags(text, opts={})
         %{#{pre}<a class="tag" href="/tags/#{url_bit}">#{clickable}</a>}
       }.html_safe
     end
+
+    def self.format_tags_for_mail(text)
```

Comment:
Assignment Branch Condition size for format_tags_for_mail is too high. [17.75/15]